lua/metals/config.lua

```diff
@@ -75,7 +75,13 @@ end
 local function check_for_coursier()
   -- this must be first the second "cs" is also found as binary but we should be more specific on Windows
   if util.is_windows then
-    return "cs.bat"
+    if util.has_bins("cs.bat") then
+      return "cs.bat"
+    else
+      log.error_and_show("Warning:Could not find cs.bat in PATH")
+      log.error_and_show("Installation instructions for Coursier https://get-coursier.io/docs/cli-installation#windows")
+      log.error_and_show("Make sure to run cs.exe once to install Coursier and add C:/Users/YOURNAME/AppData/Coursier/data/bin to your path.")
```

Comment:
Sorry if I wasn't clear in the last comment, but let's remove this line. Since it won't show correctly it's better not to have it. Also you'll need to ran `make format` to ensure everything is formatted correctly.

Reply:
> Sorry if I wasn't clear in the last comment, but let's remove this line. Since it won't show correctly it's better not to have it. 

Now, I am confused ;).
I planned to replace the three log calls with a single one and put the text in `messages.lua`. And the message would be what I suggest above. Is it ok ? I suggested the text here before commiting it to avoid polluting the logs.

> Also you'll need to ran `make format` to ensure everything is formatted correctly.

I installed stylua and ran it but I don't know if it works because it does not give me any output.

Reply:
Ach! Sorry, lol that was my mistake. Yes, your proposed text was fine. I totally misread the whole thing 😬 .

Reply:
Haha no worries. I'll do another pr right away.
BTW I am wondering how do you manage the dev and usage of nvim.metals.
Say you want to use it regularly but also want to hack on it from time to time. For example in my case I installed scalameta/metals, but to hack on it I need to pull and push to my github account. As I don't really know how packer manages this I did not swap it yet with my github account.

Reply:
Good question. What I'd recommend is just cloning this repo down on your local machine where you want to work on it, and then instead of installing `ckipp01/nvim-metals`, just install it from that local place on your machine. You can see how I do this with packer in my dot files [here](https://github.com/ckipp01/dots/blob/35ef993338165a2ea7d22a508415eb11eae63cb4/nvim/.config/nvim/lua/mesopotamia/plugins/init.lua#L32-L35)

Reply:
Great thx, I just did a pull request, I just had an issue with make format, because it would change all line endings. I was able to catch it but it might be good to add a rule for this in the Makefile. I'll look into it later.